library/src/main/kotlin/build/buf/connect/Codec.kt

```diff
@@ -39,6 +39,15 @@ interface Codec<E> {
      */
     fun serialize(message: E): Buffer
 
+    /**
+     * Deterministic Serialization of the input message.
```

Comment:
```suggestion
     * Deterministic serialization of the input message.
```
May be worth mentioning why this is useful